www-misc/zoneminder/zoneminder-1.34.8.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# TO DO:
+# * dependencies of unknown status:
+#	dev-perl/Device-SerialPort
+#	dev-perl/MIME-Lite
+#	dev-perl/MIME-tools
+#	dev-perl/PHP-Serialization
+#	virtual/perl-Archive-Tar
+#	virtual/perl-libnet
+#	virtual/perl-Module-Load
```

Comment:
So what's the deal with this list? I see these packages exist in ::gentoo repo, are they needed or not?

Reply:
Good question. This came from an ancient (and currently broken) ebuild for the previous version. Not sire what to do with it now. Shall I kill the comment?

Reply:
If possible you could check if the program can somehow optionally load these modules if they are present. But if nothing makes sense, just kill it.

Reply:
They all are referenced in code, at least. My perl knowledge is extremely limited so I'll add them as dependencies.

Reply:
> They all are referenced in code, at least. My perl knowledge is extremely limited so I'll add them as dependencies.

If you are a good ebuild comitter, you prepare a absolute minimal gentoo in a virtual machine and try to emerge your ebuild. Then you will discover all dependencies.
If you dont do this, expect some bugs in https://bugs.gentoo.org/.
As a rule of thumb, do all you can do to get a good user experience from your ebuild, like you expect from all the other ebuilds around.
There is only 3% knowledge of any programming language necessary when figuring out dependencies.